test/Microsoft.ML.IntegrationTests/Validation.cs

```diff
@@ -28,7 +29,7 @@ public Validation(ITestOutputHelper output) : base(output)
         /// pipelines on in-fold data, (3) scoring on the out-fold data, (4) returning the set of
         /// metrics, trained pipelines, and scored test data for each fold.
         /// </summary>
-        [Fact]
+        [NativeDependencyFact("MklImports")]
```

Comment:
What here depends on MklImports?

Reply:
Here we are using the Ols regression trainer which uses it:
https://github.com/dotnet/machinelearning/blob/1b3cb77b9752fe4279376039ee20fc42822e4845/src/Microsoft.ML.Mkl.Components/OlsLinearRegression.cs#L281